recommenders/models/rbm/rbm.py

```diff
@@ -691,3 +699,27 @@ def predict(self, x):
         vp = self.sess.run(v_, feed_dict={self.vu: x})
 
         return vp
+
+    def save(self, dir_name):
+        """Save model parameters in `dir_name`
+        Args:
+            dir_name (str): directory name, which should be a folder name instead of file name
+                we will create a new directory if not existing.
+        """
+        # save trained model
+        if not os.path.exists(dir_name):
+            os.makedirs(dir_name)
```

Comment:
Minor proposal, to me this is more elegant: `os.makedirs(work_directory, exist_ok=True)` and you save one line of code

Reply:
Done.